src/index.ts

```diff
@@ -5,68 +5,73 @@
  */
 
 import program = require('commander');
+import docsBaseImport = require('./docs');
+import validatorBaseImport = require('./validator');
+
 require('colors');
+const util = require('util');
 let version = require('../package').version;
-let firstArg: string | undefined = undefined
-let secondArg: string = undefined!;
 
 function list(val: string) {
     return val.split(',');
 }
 
-program
-    .version(version)
-    .arguments('<cmd> <file>')
-    .option('-p, --parameters <items>', 'List of params', list)
-    .option('-p, --pseudo <items>', 'List of pseudo overrides', list)
-    // https://github.com/tj/commander.js/issues/108
-    // might get fixed by https://github.com/tj/commander.js/issues/691
-    // as a workaround, we can actually leave this out. It defaults to true and the unparsed parameter will be ignored.
-//  .option('--guess-parameters', 'Guess any parameters that are not explicitely passed in and have no Default. This is the default behaviour.')
-    .option('-G, --no-guess-parameters', 'Fail validation if a parameter with no Default is not passed')
-    .option('-g, --only-guess-parameters <items>', 'Guess the provided parameters, and fail validation if a parameter with no Default is passed', list)
-    .option('-v, --verbose', 'Verbose error messages')
-    .action(function (arg1, arg2) {
-        firstArg = arg1;
-        secondArg = arg2;
-    });
-
-
-program.parse(process.argv);
+function doNoCommand() {
+    console.error('\nno command given!');
```

Comment:
Lets format this nicely

Reply:
Sure, but I'm not really sure what you're referring to? :)

Reply:
Sorry, can we have better English on these feedback messages? As you mentioned in the issue